README.md

```diff
@@ -22,14 +22,15 @@ Note, `yarn serve` will also hot reload for development.
 
 ## Development
 ### Frequently used yarn commands
-| Command                        | Description                                                     |
-|--------------------------------|-----------------------------------------------------------------|
-| yarn install                   | Install dependencies                                            |
-| yarn serve                     | Serve the app, and provide hot reload for development           |
-| yarn build-bundle              | Compiles and minifies for production & for other consumer apps  |
-| yarn update-flow-runner        | Interactive update for flow-runner                              |
-| yarn deep-update-flow-packages | Making sure the flow project dependencies are updated correctly |
-| yarn lint                      | Lints and fixes files                                           |
+| Command                         | Description                                                     |
+|---------------------------------|-----------------------------------------------------------------|
+| yarn install                    | Install dependencies                                            |
+| yarn serve                      | Serve the app, and provide hot reload for development           |
+| yarn build-bundle               | Compiles and minifies for production & for other consumer apps  |
+| yarn resolve-conflicting-builds | Resolves dist/ conflicts and rebuilds the application           |
+| yarn update-flow-runner         | Interactive update for flow-runner                              |
+| yarn deep-update-flow-packages  | Making sure the flow project dependencies are updated correctly |
+| yarn lint                       | Lints and fixes files                                           |
```

Comment:
I can't see what is the difference 😅 could you comment please?

Reply:
Oh, whitespace made if confusing – it's https://github.com/FLOIP/flow-builder/pull/448/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R30

Reply:
thanks for adding that new line @object-required